### DISCUSSION TITLE
Por onde começar...

### DISCUSSION THREAD

**Discussion by Leprechaun-hue** [frontendbr/forum#2364] [Perguntas e Respostas] [answered]
Por onde vocês começaram?

**Comment by LucySuxo** [comment, marked as answer]
Comecei por bootcamp (porém já tinha uma base do técnico que fiz)

**Comment by felquis** [comment]
Se você quer começar na web no front-end fazendo layout de sites,

- comece estudando para que serve a tag `html`,
- e quais são os atributos mais usados na tag `html`
- depois entenda para que serve a tag head e como é mais utilizado e para que serve

E por ai vai haha, quando passei a entender HTML melhor depois de um bom tempo fazendo layouts no Dreamweaver design mode,
foi assim que comecei a me aprofundar de como os códigos do HTML funcionavam.

----

No JavaScript, recomendo o livro JavaScript use a cabeça, que é um JavaScript meio antigo, mas me ajudou muito no inicio, quando eu só usava um pouco de jQuery.

---

Use algum curso gratuito no YouTube para fazer algum projeto básico do inicio ao fim, que vc pode ir copiando tudo que o instrutor fala, isso ajuda bastante.

---

Boa sorte!

**Comment by rafgam10** [comment]
Aprender Html e aprender um pouco de logica de programação. Se puder entender um coisa sobre Html semântico seria bom.
 
Obs: vídeos não sobre c++ ou java não de logica, vídeos de logica de programação como exemplos vídeos do Dev Aprender e Curso em Vídeo se for aprender Js(JavaScript), tem site que ajudar nisso como HackRank e também Uri. 

Bom estudos ;)

**Comment by ninetails** [comment]
Como a pergunta é bem subjetiva..........

Eu comecei instalando foruns e fazendo sites de Anime.... :)

E brincava fazendo sites na Geocities/HPG.

Mas isso há muito tempo atrás rs

**Reply by rafgam10** [reply]
eu faço isso as vezes para aprender animatição no html e css é muito :)
